src/lib/dom.generated.d.ts

```diff
@@ -7014,12 +7014,10 @@ interface ImageData {
     data: number[];
     height: number;
     width: number;
+    new(width: number, height: number): ImageData;

```

Comment:
I do not think this is correct.. we need a new interface ImageDataConstructor so that:

```ts
// ImageData stayes the way it was

 interface ImageData {
     data: number[];
     height: number;
     width: number;
}

interface IamageDataConstructor {
    prototype: ImageData;
    new (): ImageData;
    new(width: number, height: number): ImageData;
}

declare var ImageData: ImageDataConstructor; 
```